src/synchro.rs

```diff
@@ -610,9 +624,10 @@ macro_rules! resampler_FftFixedin {
 
                 let nbr_chunks_ready =
                     (self.saved_frames as f32 / self.fft_size_in as f32).floor() as usize;
```

Comment:
```suggestion
                    (self.saved_frames as f32 / self.fft_size_in as f32).floor() as usize;
                debug_assert!(wave_out.len() == self.nbr_channels);
```

Reply:
Just a heads-up on this PR. There have been quite a few changes to the library after this was opened. It will need quite some changes to fit the current version, so not much point in looking in the finer details at the moment.

Reply:
Shall I start this over from the current master branch?

Reply:
FWIW GitHub does not show any merge conflicts for this PR.

Reply:
Strange, I see large conflicts in several files. Don't know which is easier, to update this PR or to start over and just borrow ideas from it. I would probably do the latter.